packaging/neo4j-desktop/src/main/java/org/neo4j/desktop/runtime/DatabaseActions.java

```diff
@@ -105,16 +97,21 @@ public void stop()
         }
     }
 
-    public void shutdown()
+    private boolean isRunning()
     {
-        if ( isRunning() )
-        {
-            stop();
-        }
+        return server != null;
     }
 
-    public boolean isRunning()
+    private Set<Class> extractCauseTypes( Throwable e )
```

Comment:
I didn't write this part myself. I could take a look at it later if you think it could be improved.